config-server/src/main/resources/configurations/api-gateway.yml

```
@@ -0,0 +1,30 @@
+          - id: order-server
+            uri: lb:http://ORDER-SERVER
+            predicates:
+              - Path=/api/v1/orders/**
+          - id: order-line-server
+            uri: lb:http://ORDER-SERVER
+            predicates:
+              - Path=/api/v1/order-lines/**
+          - id: payment-server
+            uri: lb:http://PAYMENT-SERVER
+            predicates:
+              - Path=/api/v1/payments/**
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)